elasticapm/instrumentation/register.py

```diff
@@ -70,6 +70,7 @@
     "elasticapm.instrumentation.packages.kafka.KafkaInstrumentation",
     "elasticapm.instrumentation.packages.grpc.GRPCClientInstrumentation",
     "elasticapm.instrumentation.packages.grpc.GRPCServerInstrumentation",
+    "elasticapm.instrumentation.packages.grpc.GRPCAsyncServerInstrumentation",
```

Comment:
Could you move this to the list below that is gated by the Python 3.7 version check? We don't support asyncio tracing in Python 3.6.

Reply:
Sure, done!